app/services/auth.service.js

```diff
@@ -104,20 +103,20 @@ function setSessionVersion (req) {
 
 function redirectToLogin (req, res) {
   req.session.last_url = req.originalUrl
-  logger.info(`[${req.correlationId}] Redirecting attempt to access ${req.originalUrl} to ${paths.user.logIn}`)
+  logger.info(`Redirecting attempt to access ${req.originalUrl} to ${paths.user.logIn}`)
   res.redirect(paths.user.logIn)
 }
 
 function hasValidSession (req) {
   const isValid = sessionValidator.validate(req.user, req.session)
-  if (!isValid) logger.info(`[${req.correlationId}] Invalid session version for user. User session_version: ${lodash.get(req, 'user.sessionVersion', 0)}, session version ${lodash.get(req, 'session.version')}`)
+  if (!isValid) logger.info(`Invalid session version for user. User session_version: ${lodash.get(req, 'user.sessionVersion', 0)}, session version ${lodash.get(req, 'session.version')}`)
   return isValid
 }
 
 function addUserFieldsToLogContext (req, res, next) {
   if (req.user) {
     addLoggingField(USER_EXTERNAL_ID, req.user.externalId)
-    addLoggingField('is_internal_user', req.user.internalUser)
+    addLoggingField('internal_user', req.user.internalUser)
```

Comment:
This affects few selfservice Splunk dashboards

Reply:
we inconsistently have `is_internal_user` and `internal_user` for different log lines, so I picked the one we used for the transaction search log lines. Good point though, I'll check if any dashboards need updating.

Reply:
I'll keep it as `internal_user` and update the dashboards